build/buildHelpers.js

```diff
@@ -0,0 +1,202 @@
+const webpack = require("webpack");
```

Comment:
Everything except the parallel stuff at the bottom was just copy/pasted from gulpfile in an effort to slim that file down.